src/main/java/org/broadinstitute/hellbender/utils/fermi/FermiLiteAssembler.java

```diff
@@ -0,0 +1,161 @@
+package org.broadinstitute.hellbender.utils.fermi;
```

Comment:
I might change this package just org.broadinstitute.fermi.  It seems strange that it's in hellbender.utils.